Source/santad/EventProviders/EndpointSecurityTestUtil.mm

```diff
@@ -0,0 +1,120 @@
+/// Copyright 2021 Google Inc. All rights reserved.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+///    Unless required by applicable law or agreed to in writing, software
+///    distributed under the License is distributed on an "AS IS" BASIS,
+///    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+///    See the License for the specific language governing permissions and
+///    limitations under the License.
+#import <Foundation/Foundation.h>
+
+#include <EndpointSecurity/EndpointSecurity.h>
+#include <bsm/libbsm.h>
+
+#import "Source/santad/EventProviders/EndpointSecurityTestUtil.h"
+
+es_string_token_t MakeStringToken(const NSString *s) {
+  return es_string_token_t{
+    .data = [s UTF8String],
+    .length = [s length],
+  };
+}
+
+@implementation ESResponse
+@end
+
+@interface MockEndpointSecurity () {
+}
+@property(nonatomic, strong) NSMutableArray<void (^)(ESResponse *)> *responseCallbacks;
+@property(nonatomic) void *client;
+@property(nonatomic) es_handler_block_t __strong handler;
+@end
+
+@implementation MockEndpointSecurity
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    self.responseCallbacks = [NSMutableArray array];
```

Comment:
`_responseCallbacks = [NSMutableArray array];`

Ref: https://google.github.io/styleguide/objcguide.html#avoid-messaging-the-current-object-within-initializers-and--dealloc